core/domain/story_domain.py

```diff
@@ -343,6 +343,12 @@ def validate(self):
                 'Chapter thumbnail background color %s is not supported.' % (
                     self.thumbnail_bg_color))
 
+        if self.thumbnail_bg_color and self.thumbnail_filename is None:
+            raise utils.ValidationError(
+                'Chapter thumbnail filename is not specified.')
```

Comment:
Does this eventually get shown to the user? If so, then may need to reframe as "Chapter thumbnail image is not provided" since the user won't know anything about the filenames. Ditto elsewhere.

(If it only is for the logs then the current text is fine.)